modules/users/server/routes/admin.server.routes.js

```diff
@@ -14,13 +14,17 @@ module.exports = function (app) {
   app.route('/api/users')
     .get(adminPolicy.isAllowed, admin.list);
 
+  app.route('/api/unapproved')
+    .get(adminPolicy.isAllowed, admin.unapprovedList)
+    .post(adminPolicy.isAllowed, admin.changeToAccepted)
+    .delete(adminPolicy.isAllowed, admin.deleteApplicant);
```

Comment:
all of these methods should be on the app.route('/api/unapproved/:userId') route because there is middleware that will automatically grab the user for us based on that Id. Look at how the code block bellow does it.

Reply:
You're probably right but I have it working this way and I don't want to have to rethink the approach.

Reply:
Actually I don't think they should be anyway unless there's a convenient way to generate a request from a separate URL independently. It's not like the other User CRUD where there's a separate page/parameter for every user: all of these approval or denials come from the same view and the same url.

Reply:
what do you mean?